spec/yaml/swiftnav/sbp/ssr.yaml

```diff
@@ -920,3 +920,57 @@ definitions:
            size_fn: n_sats
            map_by: sat_id
            desc: Orbit and Clock Bounds per Satellite
+
+ - CodeAndPhaseBiasesSatelliteSignal:
```

Comment:
Any way to shorten this name?

Reply:
I can go with `CodePhaseBiasesSatSig`. With a previous name I hit the limit, so its a good idea to keep it short.

Reply:
👍